Library/Homebrew/cask/lib/hbc/source/path_base.rb

```diff
@@ -1,28 +1,72 @@
 require "rubygems"
-require "hbc/cask_loader"
```

Comment:
Did you mean to merge `CaskLoader` back into `PathBase`, or is this branch just based on an old commit?